integration/helper_test.go

```diff
@@ -231,6 +231,78 @@ var _ = Describe("gpbackup_helper end to end integration tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 			assertNoErrors()
 		})
+		It("gpbackup_helper will not error out when plugin writes something to stderr", func() {
+			setupRestoreFiles("", true)
+
+			err := exec.Command("touch", "/tmp/GPBACKUP_PLUGIN_LOG_TO_STDERR").Run()
+			Expect(err).ToNot(HaveOccurred())
+
+			go func() {
+				for _, i := range []int{1, 3} {
+					contents, _ := ioutil.ReadFile(fmt.Sprintf("%s_%d", pipeFile, i))
+					Expect(string(contents)).To(Equal("here is some data\n"))
+				}
+			}()
+
+			args := []string{
+				"--toc-file", tocFile,
+				"--oid-file", oidFile,
+				"--pipe-file", pipeFile,
+				"--content", "1",
+				"--single-data-file",
```

Comment:
Is it possible to write an intergration test to cover "--resize-cluster" option? Currently both tests launch helper with "--single-data-file" option.

Reply:
There already are end-to-end tests that use the DDP plugin with `--resize-restore`:
https://github.com/arenadata/gpbackup/blob/ab19bffa9adb32bb8013bb4ba099704722bcb06b/end_to_end/end_to_end_suite_test.go#L2158-L2163